mozjs-sys/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mozjs_sys"
 description = "System crate for the Mozilla SpiderMonkey JavaScript engine."
 repository.workspace = true
-version = "0.128.9-1"
```

Comment:
we do not actually need to do bump here (because there is no C/C++ changes), but nothing is wrong if we do.